### PR TITLE
Fix deprecated `plarform` string

### PR DESCRIPTION
**Pull request by kaakaa**
#### Summary
Fix deprecated `platform` string in `i18n/en.json`

#### Ticket Link
N/A

#### Checklist
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -1109,7 +1109,7 @@
...
   "admin.service.listenAddress": "Listen Address:",
-  "admin.service.listenDescription": "The address and port to which to bind and listen. Specifying \":8065\" will bind to all network interfaces. Specifying \"127.0.0.1:8065\" will only bind to the network interface having that IP address. If you choose a port of a lower level (called \"system ports\" or \"well-known ports\", in the range of 0-1023), you must have permissions to bind to that port. On Linux you can use: \"sudo setcap cap_net_bind_service=+ep ./bin/platform\" to allow Mattermost to bind to well-known ports.",
+  "admin.service.listenDescription": "The address and port to which to bind and listen. Specifying \":8065\" will bind to all network interfaces. Specifying \"127.0.0.1:8065\" will only bind to the network interface having that IP address. If you choose a port of a lower level (called \"system ports\" or \"well-known ports\", in the range of 0-1023), you must have permissions to bind to that port. On Linux you can use: \"sudo setcap cap_net_bind_service=+ep ./bin/mattermost\" to allow Mattermost to bind to well-known ports.",
```
Comment (jasonblais) [inline i18n/en.json]: You'll also need to update https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/admin_definition.jsx#L166

File: i18n/en.json [hunk 2]
```diff
@@ -1955,7 +1955,7 @@
   "help.formatting.headings2Example": "Large Heading\n-------------",
   "help.formatting.headingsExample": "## Large Heading\n### Smaller Heading\n#### Even Smaller Heading",
   "help.formatting.images": "## In-line Images\n\nCreate in-line images using an `!` followed by the alt text in square brackets and the link in normal brackets. Add hover text by placing it in quotes after the link.",
-  "help.formatting.imagesExample": "![alt text](link \"hover text\")\n\nand\n\n[![Build Status](https://travis-ci.org/mattermost/platform.svg?branch=master)](https://travis-ci.org/mattermost/platform) [![Github](https://assets-cdn.github.com/favicon.ico)](https://github.com/mattermost/platform)",
+  "help.formatting.imagesExample": "![alt text](link \"hover text\")\n\nand\n\n[![Build Status](https://travis-ci.org/mattermost/mattermost-server.svg?branch=master)](https://travis-ci.org/mattermost/mattermost-server) [![Github](https://assets-cdn.github.com/favicon.ico)](https://github.com/mattermost/mattermost-server)",
```
Comment (jasonblais) [inline i18n/en.json]: Same for 
 - https://github.com/mattermost/mattermost-webapp/blob/master/components/help/components/formatting.jsx#L41
 - https://github.com/mattermost/mattermost-webapp/blob/master/components/help/components/formatting.jsx#L43

File: i18n/en.json (hunk 1 above)
Comment (kaakaa) [reply i18n/en.json]: Thanks! Fixed it.

File: i18n/en.json (hunk 2 above)
Comment (kaakaa) [reply i18n/en.json]: Fixed 👍

**Comment by kaakaa** [comment]
Rebased.
Sorry for my late action.

**Comment by cpanato** [comment]
Hello @kaakaa thanks for your PR, can you please rebase so I can merge it?

thanks!

**Comment by kaakaa** [comment]
@cpanato Rebased

**Pull request merged**